Sources/PaymentButtons/PaymentButtonColor.swift

```diff
@@ -23,7 +23,7 @@ public enum PaymentButtonColor: String {
     var color: UIColor {
         switch self {
         case .gold:
-            return UIColor(hexString: "#FFC439")
```

Comment:
It would be better to include this change in https://github.com/paypal/paypal-ios/pull/243, if that's alright

Reply:
Sounds good, I've included it in #243